api/src/pcapi/routes/pro/offers.py

```diff
@@ -198,6 +198,20 @@ def patch_offer(offer_id: str, body: offers_serialize.PatchOfferBodyModel) -> of
     return offers_serialize.OfferResponseIdModel.from_orm(offer)
 
 
+@private_api.route("/offers/educational/<offer_id>", methods=["PATCH"])
+@login_required
+@spectree_serialize(response_model=offers_serialize.OfferResponseIdModel)  # type: ignore
+def patch_educational_offer(
+    offer_id: str, body: offers_serialize.PatchEducationalOfferBodyModel
+) -> offers_serialize.OfferResponseIdModel:
+    offer = load_or_404(Offer, human_id=offer_id)
```

Comment:
Toujours pas aussi fan des `load_or_404` haha
Il vaudrait mieux avoir un `get_educational_offer` qui filtre par id d'offre :) 

Et aussi pouvoir renvoyer une 404 avec un message genre `"{offerId: No educational offer has been found for this id}"`

Reply:
En faisant try except plutôt que de raise la 404 dans une méthode qui devrait servir à requêter la BDD